src/Appwrite/Platform/Tasks/Upgrade.php

```diff
@@ -23,10 +23,11 @@ public function __construct()
             ->param('image', 'appwrite', new Text(0), 'Main appwrite docker image', true)
             ->param('interactive', 'Y', new Text(1), 'Run an interactive session', true)
             ->param('no-start', false, new Boolean(true), 'Run an interactive session', true)
-            ->callback(fn ($httpPort, $httpsPort, $organization, $image, $interactive, $noStart) => $this->action($httpPort, $httpsPort, $organization, $image, $interactive, $noStart));
+            ->param('database', 'mariadb', new Text(0), 'Database to use (mariadb|postgresql)', true)
```

Comment:
We shouldn't support changing db during upgrade, we have no way to migrate existing data at the moment